server/delphilint-server/src/main/java/au/com/integradev/delphilint/remote/sonarqube/SonarQubeHost.java

```diff
@@ -191,6 +192,34 @@ public Map<String, String> getRuleNamesByRuleKey() throws SonarHostException {
     return ruleMap;
   }
 
+  private boolean getIsTaxonomyCleanCode() throws SonarHostException {
```

Comment:
Might be overkill, but should this just be `getTaxonomy` with a taxonomy enum?

Reply:
I guess I can make it an enum. It just seemed a bit unnecessary considering it's a single totally internal routine whose routine value is only used as a conditional check.

Reply:
Done.